prow/flagutil/github.go

```diff
@@ -117,7 +117,7 @@ func (o *GitHubOptions) GitClient(secretAgent *secret.Agent, dryRun bool) (clien
 	}(client)
 
 	// Get the bot's name in order to set credentials for the Git client.
-	githubClient, err := o.GitHubClient(secretAgent, dryRun)
+	githubClient, err := o.GitHubClient(secretAgent, dryRun, logrus.Fields{"component": "flagutil"})
```

Comment:
`flagutil` is not a component. We don't want to override "component: tide" just because tide uses the `flagutil` package.

Reply:
This is an ephemeral client used for one call in this helper :|

Reply:
Oh I didn't see that 🤦‍♂️